WordPress/wordpress-playground
# Show error log if Playground fails to start

Fixes #1230

## What is this PR doing?

It adds a modal that loads if Playground fails to start. The modal contains error details.

## What problem is it solving?

It should help users debug startup issues like Blueprint errors.

## How is the problem addressed?

By showing a modal with Playground logs when `startPlaygroundWeb` fails to start. 

## Testing Instructions

- Checkout this branch
- Open [Playground with an incorrect blueprint](http://localhost:5400/website-server/#{%20%22steps%22:%20[%20{%20%22step%22:%20%22error%22%20}%20]%20})
- Click _Don't show again_ 
- Reload and confirm that the modal doesn't load
